### PR TITLE
Added code for 18.1-18.4 (webGL tutorials)

### PR DESCRIPTION
**Pull request by Merijn-DH**
Code taken from livestreams 110 and 111.

### CODE REVIEW

File: Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>InstanceMode</title>
```
Comment (shiffman) [inline Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/index.html]: Did you mean to have this title here?

File: Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    
+    <script src="libraries/p5.js" type="text/javascript"></script>
+    <script src="libraries/p5.dom.js" type="text/javascript"></script>
+
+    <script src="sketch.js" type="text/javascript"></script>
+
+    <style> body {padding: 0; margin: 0;} canvas {vertical-align: top;} </style>
```
Comment (shiffman) [inline Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/index.html]: This is so silly, but I generally prefer to leave this out (unless the canvas is specifically filling the page.)

File: Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/sketch.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+function setup() {
+    createCanvas(400, 300, WEBGL);
```
Comment (shiffman) [inline Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/sketch.js]: Mind using 2 spaces? 😅

**Comment by Merijn-DH** [comment]
Updated :)

**Comment by shiffman** [comment]
Amazing!

**Pull request merged**